### DISCUSSION TITLE
Rock of Pages 🎸 [Pages Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#58247] [Other Feature Feedback, Questions, & Ideas]
We’re rocking and rolling into June - there have been so many great solves in Pages lately that I can’t wait to shout out our latest group! 

📄 **Pros in Pages**: A round of applause for these community citizens 👏
* @4vanyek: Came in with the [quick fix](https://github.com/orgs/community/discussions/57355) for a 404 error - thank you! 
* @sholohov: For a self-solve that came from reading the documentation 👓. See how the documentation helped them resolve a certificate [error](https://github.com/orgs/community/discussions/56921). 

⬆️ **Level up**: Resources to take your skills to new heights ☁️
* **Host a website using GitHub Pages #Shorts:** @mishmanners [presents](https://www.youtube.com/watch?v=WqOXxoGSpbs&list=PL0lo9MOBetEG2cjOxhvHNVS4NUYiZPm6z&index=71) how to build your website with GitHub Pages. 

🧹**Housekeeping**: Community on Discussions updates! 
✅ **Mark as Answer**: Don’t forget to click the ‘Mark as Answer’ button when someone solves your question to help others who may have the same question ✅. 

Thank you for another great month Pages crew - rock on 🤘!

**Comment by mystieneko** [comment]
>     * @4vanyek: Came in with the [quick fix](https://github.com/orgs/community/discussions/57355) for a 404 error - thank you!

woah, i didn't expect to get there :)

**Reply by queenofcorgis** [reply]
@4vanyek you earned it 😍

**Comment by queenofcorgis** [comment]
It is July which means a Pages mini check-in is fresh off the press 📰 

Give it a read on [this Discussions post](https://github.com/orgs/community/discussions/61379).
